src/helpers.h

```diff
@@ -82,5 +82,6 @@ double eval_h( double theta0,
                 double alpha );
 
 
+void print_index(INDEX_TYPE & I, int level);
```

Comment:
What is this doing? Printing an index? Is there a degug flag to decide whether to print or not?

Reply:
Yes this is for debugging.